fv3core/tests/main/test_dycore_call.py

```diff
@@ -170,9 +170,9 @@ def error_func(*args, **kwargs):
         raise AssertionError("call not allowed")
 
     mock_grid = unittest.mock.MagicMock()
-    with unittest.mock.patch("fv3core.utils.global_config.get_backend", new=error_func):
+    with unittest.mock.patch("pace.util.global_config.get_backend", new=error_func):
         with unittest.mock.patch(
-            "fv3core.utils.global_config.is_gpu_backend", new=error_func
+            "pace.util.global_config.is_gpu_backend", new=error_func
```

Comment:
Make sure there's a Jira task to delete pace.util.global_config, if we're no longer using it.

Reply:
It's being used in two spots in the main tests. Once we get rid of those, we can delete `global_config`